client/reducers/index.js

```diff
@@ -2,8 +2,10 @@ import { combineReducers } from 'redux'
 
 import foods from './foods'
 import info from './info'
+import categories from './categories'
 
 export default combineReducers({
   foods,
-  info
+  info,
+  categories
```

Comment:
Can you order this so it matches the view in the [wiki](https://github.com/matai-2019/foodlist/wiki/Redux-store)